pylint/checkers/stdlib.py

```diff
@@ -494,8 +503,12 @@ def visit_call(self, node):
                 if inferred is astroid.Uninferable:
                     continue
                 if inferred.root().name == OPEN_MODULE:
-                    if getattr(node.func, "name", None) in OPEN_FILES:
+                    if getattr(node.func, "name", None) in OPEN_FILES_MODE:
                         self._check_open_mode(node)
+                    if getattr(node.func, "name", None) in {"open"} or getattr(
+                        node.func, "attrname", None
+                    ) in {"open"}:
```

Comment:
```suggestion
                    if getattr(node.func, "name", None) in ("open",) or getattr(
                        node.func, "attrname", None
                    ) in ("open",):
```
Same here. Tuples will work just fine.

Reply:
Wouldn't a comparison with the string be fine as well? Or is comparison to a tuple better performance wise?

Reply:
Same reason as above. Although a string comparison might be possible, it would require us to change the `if` statement if we ever add additional entries. It might even be a good idea to create a new constant for `("open",)`.